lib/newrelic.js

```diff
@@ -67,32 +67,36 @@ class NewRelic {
      * @param {object} defaultMetrics default metrics to include in every New Relic event
      */
     constructor(options, defaultMetrics={}) {
+        const OPENWHISK_NEWRELIC_DISABLE_METRICS = process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS || false;
```

Comment:
Rename `OPENWHISK_NEWRELIC_DISABLE_METRICS` to something else. All caps should be reserved for constants defined outside everything.